cmd/object-api-datatypes.go

```diff
@@ -117,6 +117,8 @@ type ObjectInfo struct {
 	TransitionStatus string
 	// Name of transitioned object on remote tier
 	transitionedObjName string
+	// VersionID on the the remote tier
+	transitionRemoteVersionID string
```

Comment:
I think we can drop `Remote` from member field names. Struct member field name starting with `transition` conveys that we are referring to the transitioned object. e.g `ObjectInfo.transitionedObjName` doesn't have `Remote` in its name.
```suggestion
	transitionVersionID string
```